chips/sam4l/src/usart.rs

```diff
@@ -281,11 +281,12 @@ pub struct USARTRegManager<'a> {
     clock: pm::Clock,
     rx_dma: Option<&'static dma::DMAChannel>,
     tx_dma: Option<&'static dma::DMAChannel>,
-    is_panic: bool,
 }
 
+static mut IS_PANICING: bool = false;
```

Comment:
I agree with @bradjc. This should at least have been something like an `AtomicBool` so it wouldn't require `unsafe` to use.